### PR TITLE
Bugfix for Issue #234

### PR DESCRIPTION
**Pull request by erte0011**
In reference to this comment;

https://github.com/glumpy/glumpy/issues/234#issuecomment-590427568

### CODE REVIEW

File: glumpy/app/window/backends/backend_qt5.py [hunk 1]
```diff
@@ -323,19 +323,7 @@ def _modifiers_translate( self, modifiers ):
...
             _modifiers |= window.key.MOD_SHIFT
         if QtCore.Qt.ControlModifier & modifiers:
-            _modifiers |= window.key.MOD_CONTROL
-        if QtCore.Qt.AltModifier & modifiers:
-            _modifiers |= window.key.MOD_ALT
-        if QtCore.Qt.MetaModifier & modifiers:
-            _modifiers |= window.key.MOD_META
-        return _modifiers
-
-    def _modifiers_translate( self, modifiers ):
```
Comment (rougier) [inline glumpy/app/window/backends/backend_qt5.py]: Why did you remove this function ?

**Comment by rougier** [comment]
Thanks. Just one question (see above).

File: glumpy/app/window/backends/backend_qt5.py (hunk 1 above)
Comment (erte0011) [reply glumpy/app/window/backends/backend_qt5.py]: Removed because it is a duplicate; _modifiers_translate is in there twice, and it had the exact same method body.
